var/spack/repos/builtin/packages/mumps/package.py

```diff
@@ -164,10 +164,36 @@ def install(self, spec, prefix):
 
         install_tree('lib', prefix.lib)
         install_tree('include', prefix.include)
+
+        # Create separate libexec directory tree for libraries and
+        # headers that includes a "fake MPI" header needed for
+        # sequential MUMPS builds (i.e., without MPI) when linking
+        # MUMPS to some libraries (e.g., IPOPT). This "fake MPI"
+        # header will shadow headers in MPI implementations (e.g.,
+        # MPICH, OpenMPI), so there should two separate include
+        # directories: (1) without the "fake MPI" header
+        # (prefix.include) and (2) with the "fake MPI" header
+        # (join_path(prefix.libexec, "include").
+        #
+        # TODO: Install only static libraries in libexec tree; for now,
+        # install shared libraries instead.
+        prefix_libexec_lib = join_path(prefix.libexec, 'lib')
+        install_tree('lib', prefix_libexec_lib)
```

Comment:
I left it outside for now, following the conventions in [Homebrew's MUMPS formula](https://github.com/Homebrew/homebrew-science/blob/master/mumps.rb). In the case of `+mpi`, you're right, the comment is misleading. Homebrew also creates `libexec` to store MUMPS static libraries (they build both shared and static libraries, but put them in different places).